From 5bcd94c572769b4460dc7ce2b7bd1768bebbc42a Mon Sep 17 00:00:00 2001
From: Francois Herbert <francois@herbert.org.nz>
Date: Wed, 30 Jul 2025 13:06:34 +1200
Subject: [PATCH] Patched to match haliort extension version with
 package version

---
 Pkgfile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Pkgfile b/Pkgfile
index 92c1083..6235881 100644
--- a/Pkgfile
+++ b/Pkgfile
@@ -14,7 +14,7 @@ vars:
   UTIL_LINUX_SHA512: 12cf37ab2d62d0378b16a40e0194ef7131ef1ad06737cca3f169cfc04b9da08a4233076c819b30705e8fb2c3b8d91a1d83aac4f036ce58b9cf5928f545e511a2
   ENA_VERSION: 2.15.0 # update this when updating PKGS_VERSION in Makefile
   TENSTORRENT_VERSION: 1.34 # update this when updating PKGS_VERSION in Makefile
-  HAILORT_VERSION: 4.21.0 # update this when updating PKGS_VERSION in Makefile
+  HAILORT_VERSION: 4.21.0${PKGS_TAG:+-${PKGS_TAG}} # update this when updating PKGS_VERSION in Makefile
 
   # renovate: datasource=git-tags extractVersion=^libtiprc-(?<version>.*)$ depName=git://linux-nfs.org/~steved/libtirpc
   LIBTIRPC_VERSION: 1-3-3
-- 
2.39.5 (Apple Git-154)

